gems.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+gem 'activesupport'
```

Comment:
I'm not sure `activesupport` is needed - it was used in other project for different time conversion tasks